config/development.json

```diff
@@ -70,9 +70,10 @@
 	},
 	"HYDRA_URI": "http://localhost:9001",
 	"FEATURE_IMSCC_COURSE_EXPORT_ENABLED": true,
+	"FEATURE_COMMON_CARTRIDGE_COURSE_IMPORT_ENABLED": true,
 	"NEST_LOG_LEVEL": "debug",
 	"SESSION_SECRET": "dev-session-secret",
-	"FEATURE_COURSE_SHARE": true,
+	"FEATURE_COURSE_SHARE": false,
```

Comment:
why do change this feature flag?
This is deprecated and prepared to be removed.
There is FEATURE_COURSE_SHARE_NEW - maybe you are looking for that one?

Reply:
Because since the feature was deprecated, nobody set the feature flag to false and deactivated the feature for development.

Reply:
It is already under developing, please do not change it.
https://ticketsystem.dbildungscloud.de/browse/BC-5252